website/website/assets/js/map-en.js

```diff
@@ -222,7 +222,7 @@ function buildMap(...charts){
 			  "dataset": String(selectedDataset),
 			  "featureId": String(areas),
 			  "includeSvg": true
-			  "returnHTML": false
+			  "includeHTML": false
```

Comment:
To be consistent with the other parameter I suggest:

```suggestion
			  "includeHtml": false
```